Orange/canvas/application/addons.py

```diff
@@ -597,16 +597,19 @@ def list_pypi_addons():
     packages = []
 
     for release, urls in zip(release_data, release_urls):
-        urls = [ReleaseUrl(url["filename"], url["url"],
-                           url["size"], url["python_version"],
-                           url["packagetype"])
-                for url in urls]
-        packages.append(
-            Installable(release["name"], release["version"],
-                        release["summary"], release["description"],
-                        release["package_url"],
-                        urls)
-        )
+        if release and urls:
+            # ignore releases without actual source/wheel/egg files,
+            # or with empty metadata (deleted from PyPi?).
```

Comment:
I think smaller changesets are not as important as the best possible code.
Here, however, both favour kernc's proposal since the following lines (already split over multiple lines) then do not need additional indentation.